ui/src/data_explorer/containers/DataExplorer.js

```diff
@@ -122,12 +121,13 @@ class DataExplorer extends Component {
             actions={queryConfigActions}
             timeRange={timeRange}
             activeQuery={this.getActiveQuery()}
-            initialGroupByTime={INITIAL_GROUP_BY_TIME}
+            initialGroupByTime={AUTO_GROUP_BY}
           />
           <Visualization
             views={VIS_VIEWS}
             activeQueryIndex={0}
             timeRange={timeRange}
+            templates={TEMPLATES}
```

Comment:
This doesn't create a new one each time, so if this is changed elsewhere in the app, the change might persist / conflict with state, and further, it doesn't scale to multiple items.

Reply:
Come to think of it, unless you have a good reason we shouldn't, to be safe, let's `[...TEMPLATES]` here.

Reply:
That's the beauty of the way the `:interval:` template works, its static.